package.json

```diff
@@ -61,6 +63,13 @@
   "devDependencies": {
     "@ember-decorators/babel-transforms": "^2.1.2",
     "@ember-decorators/data": "^3.0.0",
+    "@types/ember": "^3.0.25",
+    "@types/ember-qunit": "^3.4.3",
+    "@types/ember-test-helpers": "^1.0.4",
+    "@types/ember-testing-helpers": "^0.0.3",
+    "@types/ember__test-helpers": "^0.7.6",
```

Comment:
all of the `@types/ember*` should be `~` dependencies, not `^`. Read more about the versioning strategy here https://github.com/mike-north/typever